fern/changelog/2025-07-10.mdx

```diff
@@ -0,0 +1,45 @@
+---
+**2025-07-09**
+
+- Avail Turing: Avail [v2.3.1.0-rc1](https://github.com/availproject/avail/releases/tag/v2.3.1.0-rc1)
+- Bsc Mainnet: Erigon [v1.3.10](https://github.com/node-real/bsc-erigon/releases/tag/v1.3.10)
+- Polygon Mainnet: Erigon [v3.0.12](https://github.com/erigontech/erigon/releases/tag/v3.0.12)
+
+---
```

Comment:
`---` is used to mark MDX frontmatter for metadata. Frontmatter is technically YAML so this text is getting parsed as YAML even though it's not YAML, which is breaking Fern's parser causing the build to fail.